Dell/recovery_common.py

```diff
@@ -151,6 +151,45 @@ def check_family(test):
             return True
     return False
 
+def check_install_dhc_id():
+    """The function is used to detect machine's ID for install DHC flow"""
+    path = '/sys/class/dmi/id/modalias'
+    if not os.path.exists(path):
+        return False
+    with open(path, 'rb') as rfd:
+        value = rfd.read().strip()
+        if not value:
+            return False
+    for top in [ISO_MOUNT, CDROM_MOUNT]:
+        if os.path.isdir(top):
+            plat_conf=os.path.join(top, "dhc", "platform_list", "install-id.conf")
+            if not os.path.exists(plat_conf):
+                return False
```

Comment:
I think you want a `continue` here, not a `return False` in case both `ISO_MOUNT` and `CDROM_MOUNT` both happen to exist.

Reply:
Yes,  indeed I don't think this situation, maybe I need to think why I don't think this,  thanks.